contrib/sssd.sysusers

```diff
@@ -0,0 +1 @@
+u     sssd   -   "User for sssd"     /               /sbin/nologin
```

Comment:
At the beginning of the spec-file we have:
```
# define SSSD user
%if 0%{?rhel}
%global sssd_user sssd
...
```
Would it be possible to avoid copy-paste of 'sssd' user name?